src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -69,6 +78,7 @@ public Command parseCommand(String userInput) throws ParseException {
             return new HelpCommand();
 
         default:
+            logger.finer("This user input caused a ParseException: " + userInput);
```

Comment:
`This` is redundant, as you have already specified the `userInput`.

```suggestion
            logger.finer("User input caused a ParseException: " + userInput);
```
Or
```suggestion
            logger.finer("ParseException: " + userInput);
```